packages/phone-number-privacy/package.json

```diff
@@ -10,14 +10,19 @@
     "deploy": "firebase deploy --only functions",
     "clean": "tsc -b . --clean",
     "build": "tsc -b .",
-    "lint": "tslint --project ."
+    "lint": "tslint --project .",
+    "test": "jest --runInBand",
+    "copy-env-to-firebase": "firebase functions:config:set envs.private_key=$(grep PRIVATE_KEY .env | cut -d '=' -f 2-)"
   },
   "dependencies": {
+    "bls12377js-blind": "https://github.com/celo-org/bls12377js-blind#4a2b67da441fca3fab254b6d787800f465a8dcee",
```

Comment:
Use `https://github.com/celo-org/bls12377js-blind#3a5ac03` instead which has my package.json fix in the bls lib

Reply:
kk